### ISSUE TITLE
Entity in the vocabulary not recognized at first, but only after selecting the cloud suggestion

### ISSUE THREAD

**Issue by novedavi** [insideout10/wordlift-plugin#324]
While creating a new article in test.wordlift.com an entity I knew we had in our vocabulary (Apple) was suggested in its cloud version at first:
![screen shot 2016-07-28 at 16 07 06](https://cloud.githubusercontent.com/assets/18528457/17215614/c0352494-54dd-11e6-935d-ad38797b0494.png)
selecting the cloud version, the correct entity in our vocabulary was then annotated:
![screen shot 2016-07-28 at 16 07 47](https://cloud.githubusercontent.com/assets/18528457/17215654/e7dca7ec-54dd-11e6-92df-16af2db2ce78.png)

**Comment by cyberandy** [comment]
This issue has been closed and will be followed on #306

**Comment by cyberandy** [comment]
Here is another case where entities from the local vocabulary are not detected from the analysis: 

In this blog post here: http://taoroma.it/wp-admin/post.php?post=4046&action=edit the entity **mano** is properly detected but rather than being derived from the local vocabulary (where the [entity is already present](http://taoroma.it/entity/mano/)) it comes from DBPedia. See below the _cloud_ icon being displayed in the widget. This occurs before choosing the entity and saving the post. 

![image](https://cloud.githubusercontent.com/assets/837037/18350063/65927416-75d4-11e6-8eee-29cae26971c7.png)

Once the entity has been chosen the match is properly done with the entity in the vocabulary, the icon becomes local storage and the description (displayed in the widget) is  derived from the vocabulary.

**Comment by ziodave** [comment]
I am closing this one as I think it has been superseded by #344
